slackbridge/bots.py

```diff
@@ -101,6 +103,16 @@ def check_slack_rtm(self):
             channel = self.channels[message['channel']]
             return user_bot.post_to_irc('#' + channel['name'], message['text'])
 
+    def topicUpdated(self, user, channel, newTopic):
```

Comment:
I'd name this something like `update_topic` instead to stay in the same style with the rest of the methods. Same with `newTopic` -> `new_topic`.

Reply:
This is an implementation of an event handler in IRCClient that gets called whenever the topic changes or a channel is joined for the first time, so I don't think it's possible to change the name of the function. I can change the topic variable name though.

Reply:
Oh, that's awkward, I forgot that's how it works. Maybe put a comment saying that it gets called by twisted when the topic is updated so it's clearer?